src/utils/constants.ts

```diff
@@ -1,5 +1,12 @@
 import bytes from 'bytes'
 
+import brandingIcon from '@/assets/iconography/branding.png'
+import dashboardIcon from '@/assets/iconography/dashboard.png'
+import passcodeLockIcon from '@/assets/iconography/passcode-lock.png'
+import settingsIcon from '@/assets/iconography/settings.png'
+import socialMediaIcon from '@/assets/iconography/social-media.png'
+import usersGroupIcon from '@/assets/iconography/users_group.png'
+import walletIcon from '@/assets/iconography/wallet.png'
```

Comment:
Can we have these as svgs instead of pngs?

Reply:
let's leave it for now, it consumes sometime